ci/citest/acceptance-test/tests/cmd_console_test.go

```diff
@@ -21,6 +22,23 @@ func runConsoleCmd(instance_id string, t *testing.T) {
 	RunCmdAndExpectFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -- false", instance_id))
 }
 
+func runConsoleCmdWithPrivatekey(instance_id string, private_key_path string, t *testing.T) {
+	RunCmdAndReportFail(t, "openvdc", "console", instance_id, "-i", private_key_path)
+	RunCmdAndReportFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -i %s ls", instance_id, private_key_path))
```

Comment:
Why use sh here? Why not just do this?

```
RunCmdAndReportFail(t,"openvdc", "console", instance_id, "-i",  private_key_path, "ls")
```

Also you should run gofmt on this file and any other files that aren't formatted correctly. It's best to set up your editor to run it on save.

Reply:
This test function integrate to TestLXCCmdConsole_AuthenticationPubkeymethod and modified to doesn't using "sh".